### ISSUE TITLE
MySQL lockdown doesn't appear to be working.

### ISSUE THREAD

**Issue by virtadpt** [Byzantium/Byzantium#55]
I'm not retyping the whole mess from my test machine, but here's the gist of it:

"Connect to server at localhost failed."
"Can't connect to local MySQL server..."

Basically, the mysql monitor can't connect to the MySQL instance so none of the other command will run.  The problem is in the rc.setup_mysql script someplace.

**Comment by virtadpt** [comment]
Symptoms:
- MySQL root password not changed from a blank not-password.
- Test database not removed.
- Anonymous accounts not removed.
- Application databases not created.
- Application users not created.
- Grants not executed.

**Comment by virtadpt** [comment]
Slightly higher priority: the MySQL 'root'@'127.0.0.1' account still doesn't have a password.  The mysqladmin utility misses it.

**Comment by virtadpt** [comment]
Installed debugging instrumentation.  Still no dice.  Going to sleep on it and come up with come up with an alternative approach tomorrow.

**Comment by virtadpt** [comment]
Reworked rc.setup_mysql to append some queries to lockdown.sql that change the passwords on the root accounts.  This should work for now, and it'll make it easier later to do things a different way.

**Comment by virtadpt** [comment]
As it turns out, the problem was that it takes between 5 and 10 seconds for the MySQL server to start up after rc.mysql is run by rc.setup_mysql.  I added a 10 second delay and that fixed everything.  On the other hand, all the times I've reworked the MySQL lockdown system have made it somewhat easier to maintain and extend as necessary.

And now we have something to build the password randomizers on top of.
